spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinding.java

```diff
@@ -95,6 +96,20 @@ public final synchronized void stop() {
 			this.lifecycle.stop();
 		}
 	}
+	
+	@Override
+	public final synchronized void pause() {
+		if (this.lifecycle instanceof Pausable) {
+			((Pausable)this.lifecycle).pause();
```

Comment:
space after `)` (checkstyle will complain).

Reply:
it didn't but i'll fix